clustermesh-apiserver/main.go

```diff
@@ -533,14 +534,16 @@ func runServer(cmd *cobra.Command) {
 	}
 
 	go func() {
+		timer, timerDone := inctimer.New()
+		defer timerDone()
 		for {
 			ctx, cancel := context.WithTimeout(context.Background(), defaults.LockLeaseTTL)
 			err := kvstore.Client().Update(ctx, kvstore.HeartbeatPath, []byte(time.Now().Format(time.RFC3339)), true)
 			if err != nil {
 				log.WithError(err).Warning("Unable to update heartbeat key")
 			}
 			cancel()
-			<-time.After(kvstore.HeartbeatWriteInterval)
+			timer.After(kvstore.HeartbeatWriteInterval)
```

Comment:
I suppose this should be a channel receive?

```suggestion
			<-timer.After(kvstore.HeartbeatWriteInterval)
```